dvd/dvd_dump.ixx

```diff
@@ -605,12 +749,72 @@ export bool redumper_dump_dvd(Context &ctx, const Options &options, DumpMode dum
 
     SignalINT signal;
 
+    uint8_t skip_range_idx = 0;
+    bool kreon_locked = false;
     for(uint32_t s = 0; s < sectors_count;)
     {
         bool increment = true;
 
         uint32_t sectors_to_read = std::min(sectors_at_once, sectors_count - s);
 
+        if(is_xbox && !kreon_locked)
+        {
+            // skip xbox security sector ranges and L1 filler range
+            if(skip_range_idx < xbox_skip_ranges.size())
+            {
+                if(xbox_skip_ranges[skip_range_idx][0] <= s && s <= xbox_skip_ranges[skip_range_idx][1] + 1)
+                {
+                    if(s == xbox_skip_ranges[skip_range_idx][1] + 1)
+                    {
+                        if(options.verbose)
+                            LOG_R("skipped sectors: {}-{}", xbox_skip_ranges[skip_range_idx][0], xbox_skip_ranges[skip_range_idx][1]);
+
+                        skip_range_idx++;
```

Comment:
Please prefer using pre-increment. Here it's a style question but there are situations where it's important so I like to use pre-increment whenever I can and post-increment only when I have to.

Reply:
Ack, will fix throughout

Reply:
Done